rules-tests/Naming/Rector/ClassMethod/RenameParamToMatchTypeRector/Fixture/skip_used.php.inc

```diff
@@ -4,10 +4,10 @@ namespace Rector\Tests\Naming\Rector\ClassMethod\RenameParamToMatchTypeRector\Fi
 
 final class SkipUsed
 {
-    public function run(Node $nd)
+    public function run(Node $node)
```

Comment:
It was `$nd` on purpose, to show not renamed as used

Reply:
Reverted.